types/browserify/index.d.ts

```diff
@@ -37,12 +37,21 @@ interface FileOptions {
 type InputFile = string | NodeJS.ReadableStream | FileOptions;
 
 /**
- * Options pertaining to a Browserify instance.
+ * Core options pertaining to a Browserify instance, extended by user options
  */
-interface Options {
-  // Custom properties can be defined on Options.
-  // These options are forwarded along to module-deps and browser-pack directly.
+interface CustomOptions {
+  /**
+   * Custom properties can be defined on Options.
+   * These options are forwarded along to module-deps and browser-pack directly.
+   */
   [propName: string]: any;
+  /** the directory that Browserify starts bundling from for filenames that start with .. */
+  basedir?: string;
```

Comment:
Options still has `basedir` declared.
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/17303/files#diff-6c09b620943cef5bf4bf6e0f64c7719dR64
(Couldn't figure out how to annotate it directly.)